frontend/src/api.js

```diff
@@ -87,6 +87,42 @@ function query(sql) {
   });
 }
 
+function tables() {
+  return apiCall(`/tables`);
+}
+
+/* Returns an array in the form:
+[
+  {
+    "table": "refs",
+    "columns": [
+      {
+        "name": "repository_id",
+        "type": "TEXT"
+      },
+      ...
+    ]
+  },
+  ...
+]
+*/
+function schema() {
+  return tables()
+    .then(res =>
+      Promise.all(
+        res.data.map(e =>
+          query(`DESCRIBE TABLE ${e.table}`).then(tableRes => ({
```

Comment:
@carlosms sorry for digging this up, but I think we need to change this logic eventually.

Instead for 
 - API call /tables
 - then, for each table call DESCRIBE TABLE from the frontend

We should have
 - API call to /tables, so that the backend returns all this information

as that was one of the purposes of having separate API endpoint on the backend: to abstract the details of how meta-information (schema) is collected.

Reply:
I agree with you comment. I actually though of doing it that way in this PR, but didn't want to get too ahead of myself and went for the quicker approach of doing it in the frontend.

I've created a new issue with your request: https://github.com/src-d/gitbase-playground/issues/40